### PR TITLE
Algorithm setup

### PR DESCRIPTION
**Pull request by stonov**

### CODE REVIEW

File: SQL/mysql/nutrient_doses.sql [hunk 1]
```diff
@@ -0,0 +1,151 @@
+-- Calcium Data:
+INSERT INTO nutrient_doses(nutrient_id, max_age, gneder, pregnant, nursing, LB, IA, UB)
```
Comment (edwardpassagi) [inline SQL/mysql/nutrient_doses.sql]: typo on gender

File: nutrition.py [hunk 2]
```diff
@@ -144,4 +144,4 @@ def link_food_to_meal():
     # do not uncomment the below line unless you are sure of its side effects, 
     # it will clear out all the data. 
     # processDataIntoDatabase()
-    app.run()
\ No newline at end of file
+    # app.run()
```
Comment (edwardpassagi) [inline nutrition.py]: uncomment before merging with master

File: pseudoalgo.txt [hunk 3]
```diff
@@ -0,0 +1,66 @@
+def generateMeals:
```
Comment (edwardpassagi) [inline pseudoalgo.txt:1]: save this file as a .py so that syntax highlighting can be shown.

**Review by edwardpassagi — DISMISSED** [review]

File: pseudoalgo.txt (hunk 3 above)
Comment (stonov) [reply pseudoalgo.txt:1]: it's just pseudocode, and you can select the highlighting in the options at the bottom

File: SQL/mysql/nutrient_doses.sql [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
+VALUES (1092, 18, 'F', false, false, 800, 2300, 4400);
+INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
+VALUES (1092, 1000, 'M', false, false, 1200, 3400, 6000);
+INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
+VALUES (1092, 1000, 'F', false, false, 1000, 2600, 5400);
+
+-- Sodium Data:
+INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
+VALUES (1093, 18, 'IDC', false, false, 1000, 2300, 3300);
```
Comment (edwardpassagi) [inline SQL/mysql/nutrient_doses.sql:55]: nice, i dont care about their gender if they don't conform to the normal

**Review by edwardpassagi — COMMENTED** [review]
asd

**Review by edwardpassagi — APPROVED** [review]
lgtm

**Pull request merged**